setup.py

```diff
@@ -28,7 +28,7 @@ def get_install_requires():
     os.system("rm -rf dist build")
     args = {'version': get_version()}
     print("You probably want to also tag the version now:")
-    print("  git tag -a %(version)s -m 'version %(version)s'" % args)
+    print(("  git tag -a %(version)s -m 'version %(version)s'" % args))
```

Comment:
I don't think we need this change, do we?
Are you using some software to change python 2 code to python 3? (Which is allowed, don't worry, I only want to know what prompted this!)

Reply:
We do not, I used the python's  2to3 utility, sorry for not checking all the results manually.

Reply:
Can you update your commit with this change removed?